src/reporter/reporter.py

```diff
@@ -60,8 +60,16 @@ def has_value(entity, attr_name):
     attr_value = attr.get('value', None)
     attr_type = attr.get('type', None)
 
+    if attr_value is None:
+        return False
+
+    # GH Issue #145
+    if attr_type in (NGSI_DATETIME, NGSI_ISO8601):
+        return attr_value.strip() != ''
```

Comment:
i am unsure what this actually does :) is the bug about the space among ' ' ?

Reply:
That's right, the spaced empty (typically seen in payloads from IoT Agents) was not being recognised as Null and QL attempted to translate it into a timestamp.

There's another github issue aimed to solve this more robustly (like in: what if the datetime string is "foo"). But my understanding is that this was needed asap.

Reply:
indeed